src/runtime/d3d12compute.cpp

```diff
@@ -3032,6 +3109,19 @@ WEAK int halide_d3d12compute_run(void *user_context,
         }
     }
 
+    {
+        TRACE_SCOPE("pipeline barriers");
+        // TODO(marcos): avoid placing UAV barriers all the time after a dispatch...
```

Comment:
Create a GitHub issue for this TODO and insert link to it here

Reply:
Done: https://github.com/halide/Halide/issues/5309